src/doc/en/installation/conda.rst

```diff
@@ -68,18 +68,15 @@ Using conda to provide system packages for the Sage distribution
 If Conda is installed (check by typing ``conda info``), one can install SageMath
 from source as follows:
 
-  - If you are using a git checkout::
-
-      $ ./bootstrap-conda
-
   - Create a new conda environment including all standard packages
     recognized by sage, and activate it::
 
-      $ conda env create --file environment.yml --name sage-build
+      $ conda env create --file environment-3.11-linux.yml --name sage-build
       $ conda activate sage-build
 
-    Alternatively, use ``environment-optional.yml`` in place of
-    ``environment.yml`` to create an environment with all standard and optional
+    If you use a different architecture, replace ``linux`` by ``macos``.
+    Alternatively, use ``environment-optional-3.11-linux.yml`` in place of
+    ``environment-3.11-linux.yml`` to create an environment with all standard and optional
```

Comment:
I think we cannot completely eliminate mention of the files `environment-3.9.yml` etc. and the `bootstrap-conda` script. 

At least in the packaging section of the developer's guide it should be explained how to maintain the locked files, and who should do this when.

Reply:
My idea was to push the maintenance of the lock files to a github action.

Reply:
It needs to be maintainable by developers, so a bit of documentation is necessary.

Reply:
I don't agree

Reply:
Still don't really see the point, but done.